### PR TITLE
Build installer in Github Actions. Update Helm repo before installing NFS

### PR DESCRIPTION
**Pull request by yuchaoran2011**
### What changes were proposed in this pull request?
#487 

### Why are the changes needed?
To prevent accidentally making changes that would break installer build 

### Does this PR introduce any user-facing change?
No.

### How was this patch tested?
Ran the bash snippet locally. But let's see if the Github Actions work as intended.

### CODE REVIEW

**Comment by yuchaoran2011** [comment]
Well, the build failed with:
```
git clone --single-branch --branch master git@github.com:lightbend/cloudflow.git work/cloudflow-operator/git_repo
Cloning into 'work/cloudflow-operator/git_repo'...
Warning: Permanently added the RSA host key for IP address '140.82.114.4' to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.
Makefile:222: recipe for target 'helm-fetch-cloudflow-operator' failed
Failed generating YAML for installer

Please make sure you have the correct access rights
and the repository exists.
```
Any idea why it can't clone the cloudflow repo, which is public?

File: scripts/build-all.sh [hunk 1]
```diff
@@ -67,6 +83,6 @@ for prj in $PROJECTS; do
       ;;
   esac
   RETVAL=$?
-  [ $RETVAL -ne 0 ] && echo "Failure in project $prj" && exit -1
+  [ $RETVAL -ne 0 ] && echo "Failure in project $prj" && exit 1
```
Comment (yuchaoran2011) [inline scripts/build-all.sh:86]: What does a -1 return code mean? I don't think it's valid. `exit 1` will indicate the script failed.

Comment (maasg) [reply scripts/build-all.sh:86]: Wow. That rabbit hole goes deep. Looks like this comes from `C`, where `resultCode < 0 ` was an error and `resultCode >= 0` had particular 'success' meanings.

**Comment by RayRoestenburg** [comment]
@yuchaoran2011 the build fails on yaml generation.

File: scripts/build-all.sh (hunk 1 above)
Comment (yuchaoran2011) [reply scripts/build-all.sh:86]: Well, that's not what bash uses though. In bash scripts, return code of 0 means success and 1-255 indicates error.

**Comment by yuchaoran2011** [comment]
@RayRoestenburg Yes it failed to generate YAML because somehow it couldn't clone a public repo. Weird. But I saw that you skipped doing that in your other PR, we don't have to worry about it.
I'll re-open to merge the bash script return code in this PR, after I cleanup the unneeded stuff.

**Comment by RayRoestenburg** [comment]
Closed, fixed in another PR @yuchaoran2011.

**Pull request closed without merging**